### PR TITLE
Add Projects

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

**Comment by maxammann** [comment]
I'm a little bit confused by the MapTiler documentation which is referenced as third party integrations. Is that the current documenation for android/ios?

**Comment by wipfli** [comment]
MapLibre has no docs for native as far as I know. @roblabs @ntadej @birkskyum correct me if I am wrong...

**Comment by maxammann** [comment]
> MapLibre has no docs for native as far as I know. @roblabs @ntadej @birkskyum correct me if I am wrong...

In that case I would just link the MapTiler documentation as offical.

File: content/projects/maplibre-gl-geocoder/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: "maplibre-gl-geocoder"
+weight: -800
+github: https://github.com/maplibre/maplibre-gl-geocoder
+documentation:
+  - url: https://github.com/maplibre/maplibre-gl-geocoder/blob/main/API.md
+    name: "API Documentation"
+stable: true
+---
+
+A plugin to show routing directions on a MapLibre GL JS map
```
Comment (wipfli) [inline content/projects/maplibre-gl-geocoder/index.md:11]: I think it is just a geocoder.

File: content/projects/maplibre-gl-directions/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+title: "maplibre-gl-directions"
+weight: -900
+github: https://github.com/maplibre/maplibre-gl-directions
+documentation:
+  - url: https://maplibre.org/maplibre-gl-directions/api/
+    name: "API Documentation"
+stable: true
+---
+
+Geocoding for MapLibre GL JS
```
Comment (wipfli) [inline content/projects/maplibre-gl-directions/index.md:11]: Should be directions..

Comment (wipfli) [inline content/projects/maplibre-gl-directions/index.md:11]: Should be directions..

**Review by wipfli — CHANGES_REQUESTED** [review]
Hi Max. Looks nice, thanks. Please fix the geocoder-directions stuff, otherwise it looks good to me.

**Comment by maxammann** [comment]
Thanks for catching this! Should be fine now :)

**Review by wipfli — APPROVED** [review]

**Pull request merged**